CONTRIBUTING.md

```diff
@@ -58,6 +57,26 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Triaging E2E test failures
+
+When you submit a change to the Cluster API repository as set of validation jobs is automatically executed by 
+prow and the results report is added to a comment at the end of your PR.
+
+Some jobs run linters or unit test, and in case of failures, you can repeat the same operation locally using `make` 
+in oder to investigate and fix the problem (the prow logs usually provide hints about the make target you should use). 
```

Comment:
```suggestion
Some jobs run linters or unit test, and in case of failures, you can repeat the same operation locally using `make test lint-full [etc..] ` 
in order to investigate and potential issues. Prow logs usually provide hints about the make target you should use, there might be more than one command that needs to be run. 
```

Reply:
done!